libs/sdk-ui-dashboard/src/model/commandHandlers/widgets/validation/insightDrillDefinitionValidation.ts

```diff
@@ -60,11 +67,54 @@ export function validateDrillDefinition(
         throw invalidArgumentsProvided(
             ctx,
             cmd,
-            `Invalid drill origin for InsightDrillDefinition: ${stringify(drillDefinition, {
+            `Invalid drill definition: ${stringify(drillDefinition, {
```

Comment:
i think there's no need to stringify the drill definition here. just the details why the arguments are invalid is enough. note that the command failed event will include the command (which contains the drill definition verbatim)

Reply:
Fixed